packages/store-ui/src/atoms/Select/stories/Select.mdx

```diff
@@ -4,7 +4,27 @@ import Select from '../Select'
 # Default
 
 <Canvas>
-  <Story id="atoms-select--select" />
+  <Story id="atoms-select--default-select" />
+</Canvas>
+
+# Disabled
+
+<Canvas>
+  <Story id="atoms-select--disabled-select" />
+</Canvas>
+
+
+# Mutliple
+
+<Canvas>
+  <Story id="atoms-select--multiple-select" />
+</Canvas>
+
+
+# Default Value
```

Comment:
Shouldn't the name or the id here be changed?

Reply:
Yeah could be, but imo will be strange section name: `Base Value Select`. Do you agree?

Reply:
Yeah, indeed